.github/workflows/e2e.yml

```diff
@@ -35,4 +35,5 @@ jobs:
                   composer require wp-cli/i18n-command
                   npm run build
                   npx wc-e2e docker:up
+                  sleep 10
```

Comment:
@samueljseay I added this, it seemed to fix it on this PR, I have a slight feeling that the database container wasn't ready yet, and thus the first set of tests often fails.
This should be replaced with something like we have in our unit tests: https://github.com/woocommerce/woocommerce-admin/blob/main/docker/wc-admin-php-test-suite/entrypoint.sh#L4
But doing this for now.

Reply:
I've been wondering about these failures too. I suspect the frontend of the site is ready but wc-admin is not, if that's possible. The ping to the ready page is only testing the frontend.

I think the database container is ready though.

Reply:
Yeah I wondered about this too. If a sleep helps for now I'm fine with it. 

I wonder if we could wait for a 200 from the admin login page? I still don't know if thats late enough but the tests do all start with logins anyway, so surely that'd be the first thing to timeout?

Reply:
> I wonder if we could wait for a 200 from the admin login page?

Yeah this sounds like a solid approach. It does look like the failure is from waiting for the login screen. ([example](https://github.com/woocommerce/woocommerce-admin/runs/2400582483?check_suite_focus=true#step:7:1236))

Reply:
That makes sense, I will push a separate PR up for the login screen test.